apps/profiles/models.py

```diff
@@ -22,9 +22,7 @@ class Privacy(models.Model):
         _("greit å ta bilder på arrangement"), default=None, null=True
     )
 
-    user = models.OneToOneField(
-        User, related_name="privacy", on_delete=models.CASCADE, primary_key=True
-    )
+    user = models.OneToOneField(User, related_name="privacy", on_delete=models.CASCADE)
```

Comment:
Why remove the primary key here?

Reply:
I'm just trying out a few different things. Privacy in a signal actually screws up a bunch of things (some transaction and atomic blocs). Do basically just a WIP until I figure out an alternative to postsave